libs/blocks/global-navigation/base.css

```diff
@@ -111,6 +111,10 @@
   align-items: center;
 }
 
+header.global-navigation {
```

Comment:
Based on https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/global-navigation.js#L302-L305, it could still be that there is unstyled markup until the decoration methods are done. It might be worth investigating how things look in that scenario and possibly load the base styles a bit later.

Another thing to keep in mind is that the footer also loads the base styles - https://github.com/adobecom/milo/blob/stage/libs/blocks/global-footer/global-footer.js#L97. In the (relatively unlikely) case that there are some errors in loading the top navigation that would prevent it from being visible, the footer logic would also try to make it visible. This is because the base styles were intended to be a common ground for both the header and the footer, while this change brings in specific header styles.